src/base/print_trace.C

```diff
@@ -46,7 +46,11 @@
 #include "libmesh/print_trace.h"
 #include "libmesh/libmesh.h"
 
+#ifdef LIBMESH_GDB_COMMAND
```

Comment:
`#ifdef LIBMESH_GDB_COMMAND` always evaluates to `true`, it is set to the string "no" by default, so I think those lines you added are redundant.

Reply:
Thank you. Corrected